Assets/_MUTUAL/Installers/StartupPageInstaller.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Assets.CaseFile;
+using Mutual.Screens;
+using Zenject;
+
+namespace Mutual.Installers
+{
+    class StartupPageInstaller : Installer
+    {
+        public override void InstallBindings()
+        {
+            Container.Bind<Project>().AsSingle();
```

Comment:
@RemyaTP  Revisit this binding after restructuring the Project / Patient Data types?

Reply:
@KICO-SOURCE  Are we expecting to create manager class and bind it instead?